USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -281,7 +303,51 @@ public void createBookHeaders(string bookname)
 
             bookNameCount++;
         }
-
+        public void getRenderedRows(Marker input, StyleConfig config,XWPFTable parentTable)
+        {
+            XWPFTableRow tableRowContainer = parentTable.CreateRow();
+            foreach (Marker marker in input.Contents)
+            {
+                getRenderedCell(marker, config, tableRowContainer);
+            }
+        }
+        public void getRenderedCell(Marker input,StyleConfig config, XWPFTableRow parentRow)
+        {
+            StyleConfig markerStyle = (StyleConfig)config.Clone();
+            XWPFTableCell tableCellContainer = parentRow.CreateCell();
+            XWPFParagraph cellContents = tableCellContainer.AddParagraph();
+            switch (input)
+            {
+                case THMarker tHMarker:
+                    markerStyle.isBold = true;
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker, markerStyle, cellContents);
+                    }
+                    break;
+                case THRMarker tHRMarker:
```

Comment:
I'm fairly certain THR is right aligned.

Reply:
@rbnswartz Updated